lib/Config.uk

```diff
@@ -32,3 +32,7 @@ config HAVE_SYSCALL
 config HAVE_X86PKU
 	bool
 	default n
+
+config HAVE_DRUNTIME
```

Comment:
I don't think this config option should be in the main repo. With this pull request, we're targeting D for writing internal libraries.